cypress/e2e/lifetime.cy.ts

```diff
@@ -0,0 +1,64 @@
+/// <reference types="cypress" />
+export {};
+
+import bytes from 'bytes';
+
+const GENERAL_COUPON_DISCOUNT = 'IoYrRdmY';
+const DRIVE_WEB_URL = Cypress.env('DRIVE_WEB_URL');
+const API_DRIVE_URL = Cypress.env('API_DRIVE_URL');
+
+const url = ({ planId, couponCode }: { planId: string; couponCode?: string }) => {
+  return `${DRIVE_WEB_URL}/new?redirectUrl=${encodeURIComponent(DRIVE_WEB_URL + '/checkout-plan')}${encodeURIComponent(
+    `?planId=${planId}${couponCode ? '&couponCode=' + couponCode : ''}&mode=payment`,
+  )}`;
+};
+
+//Check if the buttons works properly
+describe('Lifetime page', () => {
+  describe('When the payment lifetime button is clicked', () => {
+    describe('When the plan of 2TB is clicked', () => {
+      it('Redirect to stripe checkout with the correct planId and mode', () => {
+        cy.visit('/lifetime');
+        cy.get('#priceTable').contains('Get 2TB').click();
+
+        cy.request('get', `${API_DRIVE_URL}/payments/prices`).then((response) => {
+          response.body.map((plan) => {
+            if (bytes(plan.bytes) === '2TB' && plan.interval === 'lifetime') {
```

Comment:
It is a bad practice to use ifs on tests. If something has multiple changes, should have multiple tests. If something needs some preconditions, please ensure that **before** running the tests. Here seems that you have to get the prices from an API to be able to ensure that some url is using the codes. 

Use `beforeAll` hook or something like this to get the prices before running the tests. Also, this makes the test easy to understand and maintain, specially if the endpoint changes.

Reply:
Done. I added a before() method before running the tests. Maybe I should add a wait() method to wait for the request to be made?